packages/labs/react/src/create-component.ts

```diff
@@ -145,7 +145,7 @@ export const createComponent = <I extends HTMLElement, E extends Events = {}>(
   // it's both needed in this component to get access to the rendered element
   // and must fulfill any ref passed by the user.
   type ComponentProps = UserProps & {
-    __forwardedRef?: React.Ref<unknown>;
+    __forwardedRef: React.ForwardedRef<unknown>;
```

Comment:
Nice!

Reply:
is `__forwardedRef` always defined? A user doesn't necessarily _have_ to pass a reference, no?

Reply:
The typings for `forwardRef()` and `ForwardedRef` show that it can be `null`, but not `undefined`.

Reply:
wonderful